applications/mne_analyze/plugins/sourcelocalization/sourcelocalization.cpp

```diff
@@ -115,18 +136,46 @@ QWidget *SourceLocalization::getView()
 
 QDockWidget* SourceLocalization::getControl()
 {
-    //QDockWidget* pControl = new QDockWidget(getName());
+    DISPLIB::MinimumNormSettingsView* pMinimumNormSettingsView = new DISPLIB::MinimumNormSettingsView(QString("MNESCAN/%1").arg(this->getName()));
```

Comment:
Use auto where applicable. For example here it is clear that a raw pointer is constructed:

```
auto pMinimumNormSettingsView = new DISPLIB::MinimumNormSettingsView(QString("MNESCAN/%1").arg(this->getName()));
```

Same for the pointers below